### DISCUSSION TITLE
Initializing tailwindcss on angular

### DISCUSSION THREAD

**Discussion by bsayat** [tailwindlabs/tailwindcss#10513] [Help] [answered]
Hi everyone

I'm trying to use tailwindcss on my angular project, but when write `npm install` i get error. could someone help me?

![image](https://user-images.githubusercontent.com/45095188/217217737-77e08ff5-b9e7-497f-81b1-657b6dcd22e4.png)

project deps

"dependencies": {
    "@angular/animations": "~12.2.0",
    "@angular/cdk": "^12.2.13",
    "@angular/common": "~12.2.0",
    "@angular/compiler": "~12.2.0",
    "@angular/core": "~12.2.0",
    "@angular/forms": "~12.2.0",
    "@angular/material": "^12.2.13",
    "@angular/platform-browser": "~12.2.0",
    "@angular/platform-browser-dynamic": "~12.2.0",
    "@angular/router": "~12.2.0",
    "rxjs": "~6.6.0",
    "tslib": "^2.3.0",
    "zone.js": "~0.11.4"
  },
  "devDependencies": {
    "@angular-devkit/build-angular": "~12.2.6",
    "@angular/cli": "~12.2.6",
    "@angular/compiler-cli": "~12.2.0",
    "@tailwindcss/line-clamp": "^0.4.2",
    "@tailwindcss/typography": "^0.5.9",
    "@types/jasmine": "~3.8.0",
    "@types/node": "^12.11.1",
    "autoprefixer": "^10.4.13",
    "jasmine-core": "~3.8.0",
    "karma": "~6.3.0",
    "karma-chrome-launcher": "~3.1.0",
    "karma-coverage": "~2.0.3",
    "karma-jasmine": "~4.0.0",
    "karma-jasmine-html-reporter": "~1.7.0",
    "postcss": "^8.4.21",
    "tailwindcss": "^3.2.4",
    "typescript": "~4.3.5"
  }

**Comment by wongjn** [comment]
As the error states, `@angular-devkit/build-angular` seems to want Tailwind v2, but you have specified v3. You can try upgrading your angular packages or try with `npm install --force` or `npm install --legacy-peer-deps`.

**Comment by rubenperegrina** [comment, marked as answer]
Hi Saiok,

It looks like you have conflicts with the `@angular-devkit/build-angular` version.

Try updating to the latest stable version with `ng update @angular-devkit/build-angular` and then install Tailwind.

**Reply by bsayat** [reply]
Hi there,
Yeah I updated angular version upto 15 and it works now. Thank u!!!
